eZ/Publish/Core/Persistence/Legacy/User/Gateway/DoctrineDatabase.php

```diff
@@ -25,6 +25,8 @@
  */
 final class DoctrineDatabase extends Gateway
 {
+    public const USER_TABLE = 'ezuser';
```

Comment:
Those should be defined on the `Gateway`, not implementation because Gateway itself applies to Legacy Storage only. Past implementations are a bit inconsistent, but the most recent one, regarding switch from eZc Database Handler to Doctrine shows the correct path.